bundles/org.openhab.binding.asuswrt/src/main/java/org/openhab/binding/asuswrt/internal/constants/AsuswrtErrorConstants.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2010-2023 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.asuswrt.internal.constants;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * The {@link AsuswrtErrorConstants} class defines error constants, which are
+ * used across the whole binding.
+ *
+ * @author Christian Wild - Initial contribution
+ */
+@NonNullByDefault
+public class AsuswrtErrorConstants {
+
+    /****************************************
+     * LIST OF ERROR MESSAGES
+     ****************************************/
+    public static final String ERR_HTTP_CLIENT_FAILED = "Starting 'httpClient' failed";
```

Comment:
You might consider moving these strings to the .properties file and support i18n

Reply:
Sure that's possible. But this will result in a bigger change, because i use this as short-tag calling other functions. So i had to handle it anyway (as integer or something). But i don't have time for something in next time.